planning/behavior_path_planner/src/scene_module/start_planner/start_planner_module.cpp

```diff
@@ -690,9 +690,14 @@ std::vector<Pose> StartPlannerModule::searchPullOutStartPoses(
   const auto pull_out_lane_stop_objects = utils::path_safety_checker::filterObjectsByVelocity(
     pull_out_lane_objects, parameters_->th_moving_object_velocity);
 
+  const double s_current =
+    lanelet::utils::getArcCoordinates(status_.pull_out_lanes, current_pose).length;
+  const double max_back_distance = std::clamp(
```

Comment:
@kosuke55 
my understanding is correct?
```suggestion
// Set the maximum backward distance less than the distance from the vehicle's base_link to the lane's rearmost point to prevent lane departure.
  const double max_back_distance = std::clamp(
```

Reply:
thanks for the nice comment!